packages/playground/windows/playground/Playground.vcxproj

```diff
@@ -172,8 +172,10 @@
 
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <PropertyGroup>
+    <UseDevBundle Condition="'$(Configuration)' == 'Debug'">true</UseDevBundle>
+    <UseDevBundle Condition="'$(Configuration)' != 'Debug'">false</UseDevBundle>
     <BundleCommand>
```

Comment:
this shouldn't really be up to the app, it should just be a property you inherit from Import-ing one of our props